vpc/subnets/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "environment_name" { }
+variable "vpc_cidr_block" {
+  default = "10.0.0.0/16"
+}
+variable "vpc_id" { }
+variable "internet_gateway_id" { }
+variable "private_newbits" {
+  default = 8
+}
+variable "private_netnum_offset" {
+  default = 10
+}
+variable "public_newbits" {
+  default = 8
+}
+variable "public_netnum_offset" {
```

Comment:
these four variable are confusing as hell. Anyway we could abstract this to make it a bit more user friendly?

Reply:
Not sure. Suggestions? We could turn it into a single hash for public and private. Plus, I think we should add descriptions to to all variables as a best practice.

Reply:
I like them as separate variables, but newbits and netnum dont really mean anything to me. No suggestion today. I might have an epiphany later though